app/Exceptions/Handler.php

```diff
@@ -52,6 +52,12 @@ public function report(Exception $exception)
      */
     public function render($request, Exception $exception)
     {
+        if ($exception instanceof \Spatie\Permission\Exceptions\UnauthorizedException) {
```

Comment:
@rappasoft it depend on a developer, this is where you can change behavior when `\Spatie\Permission\Exceptions\UnauthorizedException` occured, 

the `UnauthorizedException` also return 403, it already extended with `HttpException`.